lektor/admin/static/webpack.config.js

```diff
@@ -22,41 +23,54 @@ module.exports = {
     filename: '[name].js'
   },
   devtool: 'source-map',
+  optimization: {
+    splitChunks: {
+      chunks: 'all',
+      name: 'vendor'
+      /*
+      cacheGroups: {
+        styles: {
+          name: 'styles',
+          test: /\.(le|c)ss$/,
+          chunks: 'all',
+          enforce: true
+        }
+      }
+      */
```

Comment:
I think we should either remove or use this.

Reply:
Sorry, this PR is not quite done, I should have been a bit clearer in the description. But you're right of course :)

Reply:
It is possible to mark it as a draft if that's the case :)